doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -737,6 +737,143 @@ Alternatively, you may specify the max capacity for OSDs within the Ceph
 configuration file under the respective [osd.N] section. See
 :ref:`ceph-conf-settings` for more details.
 
+Global Override of Max IOPS Capacity for multiple OSDs
+------------------------------------------------------
+
+Depending on the requirement, the max IOPS capacity of multiple OSDs may be
+overridden by a global config specification. This can be done in case multiple
+OSDs are created on the same type of underlying hardware. There are two
+scenarios to consider in order to effect this change and are discussed below.
+The examples use :confval:`osd_mclock_max_capacity_iops_hdd`.
+
+Scenario 1: Config global set before deploying OSDs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a global value is set before deploying the OSDs, the benchmark test during
+OSD deployment is skipped thus ensuring that the globally set value is in effect.
+
+The config global may be set before deploying OSDs but after enabling the Ceph
+monitor(s) using:
```

Comment:
enabling Ceph Monitors with a command of the following form:

Reply:
Condensed the content to a single section. Please re-verify.